app/src/main/java/com/karhoo/karhootraveller/presentation/karhooLabs/KarhooLabsActivity.kt

```diff
@@ -18,7 +18,6 @@ class KarhooLabsActivity : AppCompatActivity() {
         toolbar.setNavigationOnClickListener { finish() }
         featureFlag = FeatureFlag(this)
         initialiseFeatureFlags()
-        guestCheckoutSwitch.setOnCheckedChangeListener { buttonView, isChecked -> guestCheckoutClicked()}
```

Comment:
Same as previous comment, was this meant to be removed?

Reply:
Well spotted. Adding it back in.